mock-apigw/main.go

```diff
@@ -24,10 +24,38 @@ import (
 	"github.com/ministryofjustice/opg-data-lpa-codes/internal/codes"
 )
 
-const paperKeyPrefix = "PAPER#"
+const (
+	paperKeyPrefix             = "PAPER#"
+	activationKeyTable         = "lpa-codes-local"
+	paperVerificationCodeTable = "data-lpa-codes-local"
+)
+
+var (
+	db  *dynamodb.Client
+	ctx context.Context
+)
+
+func setupDatabase(requestContext context.Context) error {
+	ctx = requestContext
+
+	cfg, err := config.LoadDefaultConfig(ctx)
+	if err != nil {
+		return err
+	}
+
+	cfg.BaseEndpoint = aws.String(cmp.Or(os.Getenv("LOCAL_URL"), "http://localhost:8000"))
+	db = dynamodb.NewFromConfig(cfg)
+
+	return nil
+}
 
 func handler(w http.ResponseWriter, r *http.Request) {
 	if r.URL.Path == "/_reset_database" {
+		if err := setupDatabase(r.Context()); err != nil {
```

Comment:
I'd rather have `setupDatabase` return the `*dynamodb.Client` than have a global for it

Reply:
I did have this but it just seemed to bloat the this handling code up.